examples/canvas_lights_pointlights.html

```diff
@@ -124,8 +128,9 @@
 			function animate() {
 
 				requestAnimationFrame( animate );
+				stats.begin();
 				render();
-
+				stats.end();
 			}
```

Comment:
Have a look at [this wiki article](https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style™) for info about the formatting style. In this case, the code block section.